src/python_bindings/gfd/bind_gfd_verification.cpp

```diff
@@ -0,0 +1,34 @@
+#include "bind_gfd_verification.h"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "algorithms/gfd/mining_algorithms.h"
+#include "py_util/bind_primitive.h"
+
+namespace {
+namespace py = pybind11;
+}  // namespace
+
+namespace python_bindings {
+void BindGfdVerification(pybind11::module_& main_module) {
+    using namespace algos;
+
+    auto gfd_module = main_module.def_submodule("gfd_verification");
+
+    py::class_<graph_t>(gfd_module, "Graph_t");
+
+    py::class_<Gfd>(gfd_module, "Gfd")
```

Comment:
It would be nice if a string conversion method was added here (Python name should be `__str__`), and it shouldn't be too difficult, given that there already is a `parser::graph_parser::WriteGfd` function.